shared/charts/couchdb/templates/statefulset.yaml

```diff
@@ -61,13 +61,29 @@ spec:
             - name: ERL_FLAGS
               value: "{{ range $k, $v := .Values.erlangFlags }} -{{ $k }} {{ $v }} {{ end }}"
           livenessProbe:
+{{- if .Values.couchdbConfig.chttpd.require_valid_user }}
+            exec:
+              command:
+                - sh
+                - -c
+                - curl -G --silent --fail -u ${COUCHDB_USER}:${COUCHDB_PASSWORD} http://localhost:${COUCHDB_SVC_COUCHDB_SERVICE_PORT}
```

Comment:
Where is the `COUCHDB_SVC_COUCHDB_SERVICE_PORT` variable coming from? It seems the port it's hardcoded at some places in the chart and not in others (like the httpGet check right below). I would vote for consistency across the chart.